src/components/Pagination/Pagination.tsx

```diff
@@ -2,0 +1,107 @@
+import { getNumbers } from '../../utils';
+
+interface Props {
+  items: string[];
+  ItemsPerPage: number;
+  currentPage: number;
+  setCurrentPage: React.Dispatch<React.SetStateAction<number>>;
+}
+
+export const Pagination: React.FC<Props> = ({
+  items,
+  ItemsPerPage,
+  currentPage,
+  setCurrentPage,
+}) => {
+  function creativePageContent(
+    data: string[],
+    listLegth: number,
+    selectPage: number = 1,
+  ) {
+    const arrayOfSliceData = [];
+    const index = selectPage - 1;
+
+    for (let i = 0; i < data.length; i += listLegth) {
+      arrayOfSliceData.push(data.slice(i, i + listLegth));
+    }
+
+    const pageContent = arrayOfSliceData[index];
+
+    return pageContent;
+  }
+
+  const content = creativePageContent(items, ItemsPerPage, currentPage);
```

Comment:
The function `creativePageContent` might return `undefined` if the `index` is out of bounds. Consider adding a check to ensure that `arrayOfSliceData[index]` is defined before returning it.